opwen_email_server/integration/celery.py

```diff
@@ -111,15 +111,16 @@ def _fqn(task):
     return f'{__name__}.{task.__name__}'
 
 
-celery.conf.update(
-    task_routes={
-        _fqn(register_client): {'queue': REGISTER_CLIENT_QUEUE},
-        _fqn(index_received_email_for_mailbox): {'queue': MAILBOX_RECEIVED_QUEUE},
-        _fqn(index_sent_email_for_mailbox): {'queue': MAILBOX_SENT_QUEUE},
-        _fqn(inbound_store): {'queue': INBOUND_STORE_QUEUE},
-        _fqn(written_store): {'queue': WRITTEN_STORE_QUEUE},
-        _fqn(send): {'queue': SEND_QUEUE}
-    })
+task_routes = {
+    _fqn(register_client): {'queue': REGISTER_CLIENT_QUEUE},
+    _fqn(index_received_email_for_mailbox): {'queue': MAILBOX_RECEIVED_QUEUE},
+    _fqn(index_sent_email_for_mailbox): {'queue': MAILBOX_SENT_QUEUE},
+    _fqn(inbound_store): {'queue': INBOUND_STORE_QUEUE},
+    _fqn(written_store): {'queue': WRITTEN_STORE_QUEUE},
+    _fqn(send): {'queue': SEND_QUEUE}
+}
+
+celery.conf.update(task_routes)
```

Comment:
Previously we were updating the configuration with the argument `task_routes=task_routes`. Now we're updating the configuration with only a dictionary. This is is not equivalent; the difference can be demonstrated with the following snippet:

```py
# old code, d at the end is {'foo': 1, 'bar': {'bar': 2}}
d = {'foo': 1}
d.update(bar={'bar': 2})
print(d)

# new code, d at the end is {'foo': 1, 'bar': 2}
d = {'foo': 1}
d.update({'bar': 2})
print(d)
```

After some investigation, this is causing the integration tests to fail on CI since none of the queues ever get registered with the Celery worker. The following should fix the issues and make the build pass:

```suggestion
celery.conf.update(task_routes=task_routes)
```